endpoints/openrtb2/auction.go

```diff
@@ -419,6 +423,51 @@ func validateSChains(req *openrtb_ext.ExtRequest) error {
 	return err
 }
 
+func (deps *endpointDeps) validateEidPermissions(req *openrtb_ext.ExtRequest, aliases map[string]string) error {
+	if req == nil || req.Prebid.Data == nil {
+		return nil
+	}
```

Comment:
According to issue #1621: "If eidPermissions is not an array, ignore it and emit a 400 error with a reasonable description in the body". Unless I overlooked it, are we missing this validation? Should we do:
```
if len(req.Prebid.Data.EidPermissions) == 0{
    return fmt.Errorf(`....`)
}
endpoints/openrtb2/auction.go
```